Server.Medius/Medius/MLS.cs

```diff
@@ -3085,7 +3085,7 @@ await Program.Database.AddBuddy(new BuddyDTO()
                                 }
                             case MediusUserAction.JoinedChatWorld:
                                 {
-                                    await data.ClientObject.JoinChannel(Program.Manager.GetOrCreateDefaultLobbyChannel(data.ApplicationId));
+                                    //await data.ClientObject.JoinChannel(Program.Manager.GetOrCreateDefaultLobbyChannel(data.ApplicationId));

```

Comment:
This I will make at least some Logger.Info, I do have this commented out internally and DOES make client have no channel once initially connected to MLS. Once it fetches ChannelList, it can join one or if no channels exist, make one itself.